script.js

```diff
@@ -0,0 +1,9 @@
+// eslint-disable-next-line no-unused-vars
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will enforce a robust  and cleaner code.